hardware/P1MeterBase.h

```diff
@@ -39,4 +39,12 @@ class P1MeterBase : public CDomoticzHardwareBase
 	unsigned char l_exclmarkfound;
 
 	bool CheckCRC();
+
+
+	unsigned char m_p1version;
```

Comment:
Well, it was there on several other variables. As this file appears to be included by several other source files I figured it might be a good idea to add the `p1` prefix because I don't know what the other files are doing with it - they may be extending the class and add the same variable name themselves.